sceneview_1_0_0/src/main/java/io/github/sceneview/nodes/ViewNode.kt

```diff
@@ -52,20 +58,31 @@ class ViewNode constructor(
         viewStream = ViewStream.Builder()
             .view(view)
             .build(engine, viewWindowViewManager)
+
         geometry = Plane.Builder(size = viewStream.worldSize, normal = Direction(z = 1.0f))
             .build(engine)
-        viewStream.onSizeChanged = { size ->
-            geometry.update(engine, size = size)
-        }
+
         texture = ViewTexture.Builder()
             .viewStream(viewStream)
             .build(engine)
+
         material = materialLoader.createViewMaterial(texture, unlit, invertFrontFaceWinding)
 
-        RenderableManager.Builder(geometry.submeshes.size)
-            .geometry(geometry)
-            .material(0, material)
-            .build(engine, entity)
+        val renderView: () -> Unit = {
+            if (geometry.boundingBox.size != Float3(0f)) {
+                RenderableManager.Builder(geometry.submeshes.size)
+                    .geometry(geometry)
+                    .material(0, material)
+                    .build(engine, entity)
+            }
+        }
+
+        viewStream.onSizeChanged = { size ->
+            geometry.update(engine, size = size)
+            renderView()
```

Comment:
Please call 
```
engine.renderableManager.setGeometry(renderableInstance, geometry)
```
instead of recreating the Renderable.

Reply:
Nice catch. Done 👍